microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add Fluent Responsive stylesheet

## Overview

This PR:
 - Adds the Fabric Core package which enables us to use the Fluent grid stylesheet 
 - Adds grid, rows and columns to the major components
 - fixes theme inconsistency
 
Fixes #1531  , fixes #1490 

### Demo
1. Mobile screen (light mode)
![image](https://user-images.githubusercontent.com/18407044/156183745-74b83b23-e793-416d-bfc1-b65a5cd21935.png)
2. Mobile screen (with sample queries sidebar expanded)
![image](https://user-images.githubusercontent.com/18407044/156183893-7ea1c4a4-131b-4f7c-a3c3-ac3c91de5d5c.png)

### Notes

References: 
[Getting started with Fabric Core](https://developer.microsoft.com/en-us/fluentui#/get-started/web#fabric-core)
[Fluent Layout](https://developer.microsoft.com/en-us/fluentui#/styles/web/layout)


## Testing Instructions

* Load GE
* Zoom to 200%. Observe responsiveness
* Expand/Shrink browser window and observe the layout.
* Use the Browser Developer Tools to test these different screen sizes with the different themes (dark, light, high contrast)
![image](https://user-images.githubusercontent.com/18407044/156183048-aa2e9a43-59fd-4cf2-bdfc-f22b6df36edb.png)
